### ISSUE TITLE
GNU C Library v2.32 removed sys/sysctl.h header

### ISSUE THREAD

**Issue by luspi** [libocca/occa#399]
After being deprecated for quite some time the `sys/sysctl.h` header has finally been [removed from the GNU C Library earlier this month](https://lists.gnu.org/archive/html/info-gnu/2020-08/msg00002.html). The update recently made it into my distribution and occa (as is) doesn't build anymore.

I think that header is only included in `src/tools/sys.cpp`. So far simply commenting out that include statement seems to do the trick, but I'm hesitant to make a pull request as it don't know if it causes issues on some older systems...

**Comment by jedbrown** [comment]
This also affects me and the solution worked when applied to `v1.1.0`.
```diff
diff --git i/src/tools/sys.cpp w/src/tools/sys.cpp
index 29c76699..7924e933 100644
--- i/src/tools/sys.cpp
+++ w/src/tools/sys.cpp
@@ -13,7 +13,6 @@
 #  include <sys/types.h>
 #  include <sys/stat.h>
 #  include <sys/syscall.h>
-#  include <sys/sysctl.h>
 #  include <sys/time.h>
 #  include <unistd.h>
 #  if (OCCA_OS & OCCA_LINUX_OS)
```

The build goes through on today's `main`, but I'm getting SEGV from `occa info`. I haven't tried reducing that yet.

**Comment by dmed256** [comment]
Fixed in #432, thanks for the heads up!
